packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/aspnet/integratechallengemagiclink.md

```diff
@@ -141,7 +141,7 @@ For example, if the user has previously enrolled the email authenticator, Google
 
 </div>
 
-### 5. Check authenticator status
+### 5. Submit the email authenticator
 
 When the user selects the Email Authenticator and clicks **Submit**, the form posts back to the `SelectAuthenticatorAsync` method. This checks whether the user is in Challenge or Enrollment Flow.
```

Comment:
What is "This" referring to in "This checks ..."?

Reply:
@robertnicolo-okta "This" refers to the SelectAuthenticatorAsync method. 

Changed text to "`SelectAuthenticatorAsync` method, which checks"